{{cookiecutter.project_slug}}/clients/vue3/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="page-not-found">
+    <h1 class="page-not-found__error">404</h1>
+    <img alt="Project Logo" src="../assets/logo.png" />
+    <h2 class="page-not-found__header">Oops!<br />We couldn't find the page you're looking for.</h2>
+    <p class="page-not-found__route">
+      Click <router-link to="/">here</router-link> to return to the home page.
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'PageNotFound',
+}
+</script>
+
+<style scoped lang="scss">
+.page-not-found {
+  &__error {
+    margin: unset;
+    font-weight: 800;
+    font-size: 64px;
```

Comment:
**Info:** Added an arbitrary larger font size for the "404" text just to display it larger on the page to make it more obvious to visual users that they've landed on a not-found page. Font size can be increased or decreased if needed.

Reply:
**Question:** Do we want the "404" error code to be displayed on the page at all? And if so, do we want the text to have a different color and/or style to further separate it from the rest of the text?

Reply:
We've been holding off on too much polish on these pages, largely because we wanted to discuss the design system stuff first. I think once a decision is made there an effort can be made to create a consistent default look and feel to all the default pages.